meetings/2022-06-09.md

```diff
@@ -4,16 +4,40 @@
 
 ### Attending
 
--
+- Jesse Mapel
+- Lauren Adoram-Kershner
+- Justin Hagerty
+- Adam Paquette
+- Lori Pigue
+- Stuart Robbins
+- J Laura
+- Amy Stamile
+- Kelvin Rodriguez
+- Victor Silva
+- Audrie Fennema
 
 ## Agenda / Notes
 
 - Action Items from last month
   - J/Jesse/Ross: Resolve #4792 by 5/20
+    - Merged
+    - If you have any comments or changes you would like to see please comment or open a PR
+    - API breaks require a RFC now
   - Jesse: Give temp permissions to Name Change WG members
+    - Will likely do the same thing for next group
   - Jesse: Update ISIS PR templates with new requirements
+     - Not done
+     - API break requirement
+     - Will happen across ASC projects (ALE, PLIO, etc)
 
 - ISIS Name Change Proposal
+  - Recommendation is to change the name
+    - No objects from those present
+    - Those not attending can comment on the PR until the next TC meeting
```

Comment:
@scsides @rbeyer @rfergason If you have any comments please leave them somewhere on this PR before the July meeting.

Reply:
Tagging other TC members not in attendance: @michaelaye @KrisBecker @AndrewAnnex